migrations/versions/498b78507b64_.py

```diff
@@ -0,0 +1,23 @@
+"""empty message
+
+Revision ID: 498b78507b64
+Revises: e3caa0f2a16c
+Create Date: 2019-01-31 14:35:08.791562
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '498b78507b64'
+down_revision = 'e3caa0f2a16c'
+
+
+def upgrade():
+    op.add_column(u'events',sa.Column('is_featured', sa.Boolean(), nullable=False))
```

Comment:
Are you sure this migration is working? There is no default value

Reply:
@iamareebjamal It is working in my system, but if you recommend, I can open another PR making the migrations again

Reply:
@iamareebjamal Actually, I think I will try and migrate again, I shall modify this file accordingly

Reply:
@iamareebjamal It works, I rerun the migrations and it does return it this way only

Reply:
@iamareebjamal already mentioned this. It's not working, here is the error log on heroku:
```
    exc_info
  File "/app/.heroku/python/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 203, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/app/.heroku/python/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 186, in reraise
    raise value.with_traceback(tb)
  File "/app/.heroku/python/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1182, in _execute_context
    context)
  File "/app/.heroku/python/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 470, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.IntegrityError: (psycopg2.IntegrityError) column "is_featured" contains null values
 [SQL: 'ALTER TABLE events ADD COLUMN is_featured BOOLEAN NOT NULL']
```